### PR TITLE
Create login and welcome page

### PR DESCRIPTION
**Pull request by nicot**
This commit displays a welcome message to users who are not logged in, and adds the log in page that Sam created to the front page.

It should not be merged as is. It'd be great to get a code review from the front-end team.

### CODE REVIEW

File: src/static/dist/all.js [hunk 1]
```diff
@@ -269,4 +279,77 @@ var Table = React.createClass({
...
+    displayName: "About",
+
+    render: function render() {
+        return React.createElement(
+            "div",
+            { className: "updates mdl-card mdl-shadow--2dp mdl-cell mdl-cell--8-col" },
+            React.createElement(
+                "p",
+                null,
+                "Campus Consensus is a survey and data collection system developed for improving student-faculty relationships at universities being developed at University Colorado at Boulder. Our platform allows students and faculty to ask questions and offer short surveys to improve instructional quality, align student and faculty interests, and express consensus about campus issues."
```
Comment (antsankov) [inline src/static/dist/all.js]: This looks good! Was this the info Michael gave to include in the about?

**Comment by untra** [comment]
We gots merge conflicts. @nicot fix those conflicts

**Comment by wannabeCitizen** [comment]
I'm not seeing anything that saves the response in the DB. Is that for sung? Or are you waiting?

**Comment by nicot** [comment]
https://github.com/SFII/scq/blob/master/src/handlers/survey_handler.py#L15 ->
https://github.com/SFII/scq/blob/master/src/models/basemodel.py#L104

I think this should save the response to the DB. Am I mistaken? The pull requests got pretty mixed up due to nobody responding for 2 weeks.

**Comment by wannabeCitizen** [comment]
Oh yeah, that does look like it'd save to the DB. Have you tested it?

**Pull request merged**